packages/experimental/Text/src/Variants.ts

```diff
@@ -0,0 +1,71 @@
+import { Variants } from '@fluentui-react-native/theme-types';
+import { Text } from './Text';
+import { createFontAliasTokens } from '@fluentui-react-native/win32-theme';
+
+const variantsWin32: Partial<Variants> = createFontAliasTokens('light');
```

Comment:
>variantsWin32 [](http://example.com/codeflow?start=6&length=13)

I would've expected to get this information from the theme object, is there a reason we can't do that? I think customize takes a function like:

```
Text.customize((t: Theme) => ({
  fontFamily: t.typography.variants.caption1.face,
}));
```

Reply:
If users override the typography variants on the Theme object in order to change the font styles then that won't be respected if we integrate the font alias tokens this way

Reply:
Ah you're right. I'll have it pull from Theme object. createFontAliasTokens will eventually be used to set the defaultTypography tokens for win32